akka-stream/src/main/scala/akka/stream/stage/GraphStage.scala

```diff
@@ -683,6 +683,13 @@ abstract class GraphStageLogic private[stream] (inCount: Int, outCount: Int) {
     }
   }
 
+  final def getAsyncCallback(handler: () ⇒ Unit): AsyncCallbackNoArgs = {
+    new AsyncCallbackNoArgs {
+      override def invoke(): Unit =
+        interpreter.onAsyncInput(GraphStageLogic.this, Unit, (Unit) ⇒ handler())
+    }
+  }
+
```

Comment:
no, please don’t: this harms type inference (overloading is bad) and it is unnecessary; just send `()` as the signal